### PR TITLE
Feature/ogc 1725 news subscribe for categories

### PR DESCRIPTION
**Pull request by Tschuppi81**
Newsletter: Support for newsletter category definition and subscription

An organization can define newsletter categories. Subscribing users can select their categories of interest and will only receive newsletters that report on at least one of their subscribed categories.

TYPE: Feature
LINK: ogc-1725

## Checklist

- [x] I have performed a self-review of my code
- [x] I considered adding a reviewer
- [x] I have updated the PO files
- [x] I made changes/features for both org and town6
- [x] I have added tests for my changes/features

### CODE REVIEW

File: src/onegov/org/forms/newsletter.py [hunk 1]
```diff
@@ -237,6 +241,15 @@ class NewsletterSendForm(Form):
     if TYPE_CHECKING:
         request: OrgRequest
 
+    categories = MultiCheckboxField(
+        label=_("Categories"),
+        description=_("Select categories the newsletter reports on. The "
+                      "users will receive the newsletter only if it "
+                      "reports on at least one of the categories the user "
+                      "subscribed to."),
```
Comment (BreathingFlesh) [inline src/onegov/org/forms/newsletter.py:249]: The translation for this is missing

**Review by BreathingFlesh — COMMENTED** [review]
I need some more information on how to use the categories, it's not clear to me where they can be created.

**Comment by Tschuppi81** [comment]
> more information on how to use the categories, it's not clear to me where they can be created.

In `/newsletter-settings` you can define the categories as yaml list, an example is provided

**Comment by Tschuppi81** [comment]
Open items after review:
- [ ] Initial newsletter subscription: success message with categories
- [ ] Initial newsletter subscription: email confirmation with categories
- [ ] newsletter subscription redirect to `/home` or `/news`
- [ ] prepare own view to 'edit newsletter subscription'
- [ ] remove edit bar for `send` view

**Comment by Tschuppi81** [comment]
@BreathingFlesh Would we need a navigation back to `/newsletters` once we are in `/newsletters/update`? What do you think?

**Pull request merged**